Tensile/Common.py

```diff
@@ -703,6 +703,14 @@
     "LdsPadA":                     [ -1, 0, 1, 2, 3, 4, 8],
     "LdsPadB":                     [ -1, 0, 1, 2, 3, 4, 8],
 
+    # Padding boundary for LDS. defines block-size for pad insertion. for every 'LdsBlockSizePerPad' bytes, LDS padding (pad value from LdsPad parameter)
+    # is added (readOffset aware of the pad and adjusts offset value based on this parameter value).good rule of thumb is LdsBlockSizePerPad >= unrollDepth * BPE
+    # optimized value is 128
+    "LdsBlockSizePerPad":          [-1, 64, 128, 256],
+
+    #Transpose LDS format. format to to store LDS in same memory format as global memory format. applicable only in TLU=0 case
+    "TransposeLDS":                [-1, 1, 0],
+
```

Comment:
How do we add no padding - should this allow a value of 0?
How does this interact with existing LdsPadA/B, which IIRC are pads specified in terms of elements

Reply:
transposeLDS use LdsPad parameter to determine padding size (ldsPad*BPE) to be added to lwo. transposeLDS=0 branches the code old implementation, i.e swizzle (write perp elements) global fetch elements during local write